### PR TITLE
S174 trophy room frontend

### LINKED ISSUE #200: Create trophy room page and background

### PR DESCRIPTION
**Pull request by chenga3**
image

Implemented frontend for trophy room page. Correct number of trophies render based on a hard-coded variable.

Still to do (front end):
- Add responsiveness
- Add popover for trophy info (name, description) which renders on hover

Closes #200

### CODE REVIEW

File: components/TrophyRoom/TrophyCabinet/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import Image from 'next/image'
+
+import TrophyGroup from 'components/TrophyRoom/TrophyGroup'
+
+import TrophyCabinetImage from 'public/images/trophy-cabinet.png'
```
Comment (CodeWithTheDoctor) [inline components/TrophyRoom/TrophyCabinet/index.tsx:6]: Trophy cabinet should also be .webp no?

Comment (chenga3) [reply components/TrophyRoom/TrophyCabinet/index.tsx]: donezo

**Comment by CodeWithTheDoctor** [comment]
Need to set the `Trophy Room` button in the navbar to send us to the trophy-room page

**Comment by CodeWithTheDoctor** [comment]
Noticed some styling issue we need to fix
image

Should be a lil tweaking needed in the classes added. I might work on fixing those

**Review by CodeWithTheDoctor — CHANGES_REQUESTED** [review]
Just need to fix bit of styling and It's good to be merged then

**Review by CodeWithTheDoctor — APPROVED** [review]

**Pull request merged**